presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/GeometryStateFactoryTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial.aggregation;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class GeometryStateFactoryTest
+{
+    private GeometryStateFactory factory;
+
+    @BeforeMethod
+    public void init()
+    {
+        factory = new GeometryStateFactory();
+    }
+    @Test
+    public void testCreateSingleStateEmpty()
```

Comment:
I suggest to combine `testCreateSingleStateEmpty` and `testCreateSingleStatePresent` tests (ditto for grouped state). Otherwise, `testCreateGroupedStateEmpty` needs to be modified to not include test of a non-empty state.